runtime/assembly/modules/storage.ts

```diff
@@ -40,28 +39,24 @@ export namespace Storage {
     }
     /**
      * Gets the given key from storage, placing the value into a buffer and returning the number of bytes that the
-     * entry in storage has beyond the offset. 
-     * ArrayBuffer has fixed length of 20.
-     * Returns Option instance with populated ArrayBuffer concatenated to the result from storage_read
+     * entry in storage has beyond the offset.
```

Comment:
"entry in storage has beyond the offset" -> "entry in storage has"

Reply:
I just copied the description from the Polkadot Host specs, maybe just leave it like this?